lib/base.php

```
@@ -586,9 +590,6 @@ public static function init() {
 		self::checkInstalled();
 
 		OC_Response::addSecurityHeaders();
-		if (self::$server->getRequest()->getServerProtocol() === 'https') {
-			\ini_set('session.cookie_secure', true);
```

why not silence the function call? We have many ini_set calls in the beginning of this function - let's just do it the same here ...

---

@DeepDiver1975 
Well - we might just drop it after all - as said - trying to change the behavior of a session setting, after the session has started, is not possible. 

In php < 7.2 this was just silently ignored - Just silencing it again when there is an underlying error seems quite wrong.

---

Then we better move this all into the Session class and do all setini before a Session is started.

I'll so a pr .....